ddl/db_change_test.go

```diff
@@ -861,6 +862,7 @@ func (s *testStateChangeSuite) TestParallelDDLBeforeRunDDLJob(c *C) {
 				info = is
 				break
 			}
+			log.Infof("time.Sleep(%v) in TestParallelDDLBeforeRunDDLJob", interval)
```

Comment:
Is it for debug?

Reply:
@lamxTyler @winkyao change it to fix `TestParallelDDLBeforeRunDDLJob` error.

Reply:
what error of `TestParallelDDLBeforeRunDDLJob`?

Reply:
I prefer to remove this log.

Reply:
Please address the comment.
@winkyao

Reply:
No, I add it to notify the test is blocked in this function, if we remove this log, it is really hard to find out the TestParallelDDLBeforeRunDDLJob hang up.